frontend/src/components/widgets/FileUploader/FileUploader.tsx

```diff
@@ -231,37 +228,40 @@ class FileUploader extends React.PureComponent<Props, State> {
     }
   }
 
-  private delete = (fileId: string): void => {
-    // TODO resolve typing issues
-    // @ts-ignore
-    this.setState(state => {
-      const files = [...state.files]
-      const file = files.find(file => file.id === fileId)
-      if (fileId && file) {
-        file.status = FileStatuses.DELETING
-        if (file.errorMessage) {
-          this.removeFile(fileId)
-          return null
-        }
-
-        if (file.cancelToken) {
-          // The file hasn't been uploaded. Let's cancel the request
-          // However, it may have been received by the server so let's
-          // send out a request to delete in case it has
-          file.cancelToken.cancel()
-        }
+  private setError = (errorMessage: string): void => {
+    this.setState({
+      status: FileStatuses.ERROR,
```

Comment:
Nit: Is it possible to change enum from `FileStatuses` to `FileStatus` to be consistent with our codebase?

From [StackOverflow](https://stackoverflow.com/questions/15755955/naming-of-enums-in-java-singular-or-plural):
> The thinking is that you're not selecting multiple Protocols, but rather one Protocol of the possible choices in the list of values.

Same for `FileSizes`.